cloud/endagaweb/tasks.py

```diff
@@ -439,3 +439,17 @@ def req_bts_log(self, obj, retry_delay=60*10, max_retries=432):
         raise
     finally:
       obj.save()
+
+@app.task(bind=True)
+def block_user(self):
+    """ Block  User if User password is not updated
```

Comment:
Please make the comment reflect the fact that the number of days is configured in a setting.